docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -6,11 +6,59 @@ weight: 80
 cascade:
 ---
 
-**Keptn's Lifecycle Scheduler** replaces the
-[Kubernetes scheduler](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/)
-to allow users to schedule events and tasks to occur
-at specific times during the application lifecycle.
-The Lifecycle Scheduler can trigger events such as
-deployment, testing, and remediation at specific times or intervals.
-The Keptn Scheduler guarantees that Pods are initiated only after
-the Pre-Deployment checks are completed.
+The **Keptn Scheduler** is an integral component of the Keptn Lifecycle Toolkit that orchestrates
+the deployment process.
+The **Keptn Scheduler** works by registering itself as a Permit plugin within the Kubernetes
+scheduling cycle that ensures that Pods are scheduled to a node until and unless the
+pre-deployment checks have finished successfully.
+This helps to prevent Pods from being scheduled to nodes that are not yet ready for them,
+which can lead to errors and downtime.
+Furthermore, it also allows users to control the deployment of an application based on
+customized rules that can take into consideration more parameters than what the default
+scheduler has (typically CPU and memory values).
+
+The Keptn Scheduler uses the Kubernetes
+[Scheduler Framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/) and is based on the
+[Scheduler Plugins Repository](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master)
+
```

Comment:
maybe worth mentioning that we are using a permit plugin 
https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/#permit

Reply:
Sure will add that info